middlewares/metrics.go

```diff
@@ -1,12 +1,35 @@
 package middlewares
 
 import (
+	"fmt"
 	"net/http"
+	"os"
 	"time"
 
+	"github.com/DataDog/datadog-go/statsd"
 	"github.com/prometheus/client_golang/prometheus"
 )
 
+var statsdClient *statsd.Client
+
+func initStatsdClient() {
+	statsdHost := os.Getenv("STATSD_HOST")
+	statsdPort := os.Getenv("STATSD_PORT")
+	applicationNamespace := os.Getenv("STATSD_APP_NAME")
+
+	var err error
+	statsdClient, err = statsd.New(statsdHost + ":" + statsdPort)
```

Comment:
I've gone back and forth on defaulting these. I'd like to fail fast, but there is a chance someone will be using this without dogstatsd.

Reply:
I didn't fail fast in the node implementations of these. I've also been bitten by not failing fast before. For this though, I'm ok with letting the default of "localhost:8125" be the fallback. We may change our minds later and that's ok too. We don't currently have a running statsd agent anyways, so we'd also need to have an on/off switch if we plan on failing fast.